src/Entity/Shipment.php

```diff
@@ -288,7 +287,7 @@ public function getShipFrom()
      *
      * @return Shipment
      */
-    public function setShipFrom(ShipFrom $shipFrom)
+    public function setShipFrom(ShipFrom $shipFrom = null)
```

Comment:
this is useful when re-using the shipment object for consecutive shipment requests. Which is, when i think about it, is quite rediculous and unbelievable. Will fix this